src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -253,21 +253,27 @@ class ReviewConsensusVariants
           val mi = toMi(rec)
           val consensusReadName = c.getRecord.getReadName + readNumberSuffix(rec)
           val rawCounts = BaseCounts(rawByMiAndReadNum(mi + readNumberSuffix(rec)))
-
-          val m =  ConsensusVariantReviewInfo(
+          val filters = {
+            variant.filters.getOrElse("") match {
+              case "" => "NA"
```

Comment:
I don't love this.  I usually pop these files open in excel, and it's much easier to differentiate empty vs. non-empty.

Also, if we're going to put anything here I would think `PASS` or `.` to mirror what VCF does would be clearer.

Reply:
Also, isn't this a much more verbose way of writing `variant.filters.getOrElse("NA")`?  If so, you could just do that and move it back inline.  Of it `variant.filters` sometimes contains `Some("")` then I'd say we should fix that and or do `variant.filters.filter(_.nonEmpty).getOrElse("NA")`.

Reply:
The latter is the problem I had, which was `variant.filters` sometimes returns `Some("")`, so I have changed it to return "PASS"